project/plugins.sbt

```diff
@@ -1 +1,7 @@
+resolvers += "Sonatype snapshots" at "https://oss.sonatype.org/content/repositories/snapshots/"
+
+addSbtPlugin("com.github.mpeltonen" % "sbt-idea" % "1.7.0-SNAPSHOT")
```

Comment:
This really slows the compile time because the `SNAPSHOT` suffix makes a request every time in order to fetch new changes in the plugin. Could we simply put the instructions about how to add the plugin?